client/components/addons/addon-collection/container.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react'
+import { observer, inject } from 'mobx-react'
+import PropTypes from 'prop-types'
+
+import AddonCollectionPage from './addon-collection'
+
+@inject('addon')
+@observer
+class AddonCollectionPageContainer extends Component {
+	static propTypes = {
+		addon: PropTypes.object.isRequired
+	}
+
+	render () {
+		const { addon } = this.props
+
+		return (
+			<AddonCollectionPage
+				addons={addon.addonsSortedByMostRecent}
```

Comment:
Yep, good catch. Will be needed to be able to reuse component for other orderings